source/gui/nvdaControls.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: UTF-8 -*-
+#settingsDialogs.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2006-2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+import wx
+from wx.lib.mixins import listctrl as listmix
+
+class AutoWidthColumnListCtrl(wx.ListCtrl, listmix.ListCtrlAutoWidthMixin):
+	"""
+	A list control that allows you to specify a column to resize to take up the remaining width of a wx.ListCtrl
+	"""
+	def __init__(self, parent, ID, autoSizeColumnIndex="LAST", pos=wx.DefaultPosition, size=wx.DefaultSize, style=0):

```

Comment:
wx controls usually take `id` as an optional argument defaulting to `wx.ID_ANY`. So, please change `ID` to `id=wx.ID_ANY`.